web_external/Literature/publications.bibtex

```diff
@@ -396,3 +396,702 @@
+  booktitle    = {Applied Imagery Pattern Recognition Workshop (AIPR), 2016 IEEE},
+  year         = {2016},
+  pages        = {1--7},
+  organization = {IEEE},
+  url          = {https://ieeexplore.ieee.org/abstract/document/8010594/},
+}
+
+@Article{ferris2016reply,
+  author    = {Ferris, Laura K and Satyanarayanan, Mahadev},
+  title     = {Reply to:“Computer-aided classification of melanocytic lesions using dermoscopic images: Low reported accuracy for reader study on melanomas with low melanoma in situ to invasive melanoma ratio”},
+  journal   = {Journal of the American Academy of Dermatology},
+  year      = {2016},
+  volume    = {75},
+  number    = {3},
+  pages     = {e121},
+  publisher = {Elsevier},
+  url       = {https://www.jaad.org/article/S0190-9622(16)30157-8/pdf},
+}
+
+@Article{fornaciali2016towards,
+  author  = {Fornaciali, Michel and Carvalho, Micael and Bittencourt, Fl{\'a}via Vasques and Avila, Sandra and Valle, Eduardo},
+  title   = {Towards automated melanoma screening: Proper computer vision \& reliable results},
+  journal = {arXiv preprint arXiv:1604.04024},
+  year    = {2016},
+  url     = {https://arxiv.org/pdf/1604.04024.pdf},
+}
+
+@Misc{abedini2015accurate,
+  author    = {Abedini, MANI and Chen, QIANG and Codella, NOEL CF and Garnavi, RAHIL and Sun, X},
+  title     = {Accurate and scalable system for automatic detection of malignant melanoma},
+  year      = {2015},
+  journal   = {Dermoscopy Image Analysis},
+  pages     = {293--343},
+  publisher = {Boca Raton, CRC Press},
+  url       = {https://www.researchgate.net/profile/Mani_Abedini/publication/300467789_Accurate_and_Scalable_System_for_Automatic_Detection_of_Malignant_Melanoma/links/579aa51e08ae024e100e4c99/Accurate-and-Scalable-System-for-Automatic-Detection-of-Malignant-Melanoma.pdf},
+}
+
+@Article{bansal2016stanford,
+  author = {Bansal, Devika G},
+  title  = {Stanford Medicine},
+  year   = {2016},
+  url    = {https://pdfs.semanticscholar.org/7d16/1843cdc6a92cafccbdc77b46d48901b076d3.pdf},
+}
+
+@Article{maiaanalise,
+  author = {Maia, Lucas Bezerra and Pereira, Roberto M Pinheiro and Silva, Thalles Alencar and Pessoa, Alexandre C{\'e}sar P and Junior, Geraldo Braz},
+  title  = {An{\'a}lise e Sugestao Autom{\'a}tica de C{\^a}ncer de Pele atrav{\'e}s de HOG e SVM},
+  year   = {2016},
+  url    = {http://sistemas.deinf.ufma.br/anaisjim/artigos/2016/201629.pdf},
+}
+
+@Article{kalouchevision,
+  author = {Kalouche, Simon},
+  title  = {Vision-Based Classification of Skin Cancer using Deep Learning},
+  url    = {http://web.stanford.edu/~kalouche/docs/Vision_Based_Classification_of_Skin_Cancer_using_Deep_Learning_(Kalouche).pdf},
+}
+
+@Article{fornaciali2015towards,
+  author    = {Fornaciali, Michel Silva and others},
+  title     = {Towards robust melanoma screening= a case for enhanced mid-level features= Triagem robusta de melanoma: em defesa dos descritores aprimorados de n{\'\i}vel m{\'e}dio},

```

Comment:
This is not correct (note the improper characters and concatenated translation). The proper info can be found at http://repositorio.unicamp.br/handle/REPOSIP/258945